.github/workflows/auto-cherry-pick.yml

```
@@ -0,0 +1,263 @@
+name: Auto Cherry-Pick
+on:
+  pull_request:
+    types: [closed]
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: write
+  pull-requests: write
+  issues: write
```

```suggestion
permissions: {}
```

We can default to no permissions for the entire workflow and then give each job just the perms it needs.

See: https://docs.zizmor.sh/audits/#remediation_9

---

good catch, thank you

---

nice, thank you